src/pages/FrontPage.js

```diff
@@ -43,15 +43,15 @@ export const  FrontPage = React.createClass({
             <ButtonItem color='green' onClick={() => this.displayExercise}>
               Kom i gang!
             </ButtonItem>
-            <ButtonItem color='blue' onClick={() => this.setState({show: !this.state.show})}>
+            <ButtonItem color='blue' onClick={() => this.props.changeMode()}>
```

Comment:
Enten vise tekst som du foreslår, eller kanskje fjerne begge knappene. 'Kom i gang!' er for elever, ikke lærere. Og man kan alltids gå tilbake til elevmodus i navigasjonsmenyen (som det står i teksten i dropdownboksen)